cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -726,28 +750,28 @@ public Route generateExternalRoute(int pod) {
 
             Route route = new RouteBuilder()
                     .withNewMetadata()
-                        .withName(perPodServiceName)
-                        .withLabels(getLabelsWithName(perPodServiceName, templatePerPodRouteLabels))
-                        .withAnnotations(mergeLabelsOrAnnotations(null, templatePerPodRouteAnnotations))
-                        .withNamespace(namespace)
-                        .withOwnerReferences(createOwnerReference())
+                    .withName(perPodServiceName)
+                    .withLabels(getLabelsWithName(perPodServiceName, templatePerPodRouteLabels))
+                    .withAnnotations(mergeLabelsOrAnnotations(null, templatePerPodRouteAnnotations))
+                    .withNamespace(namespace)
+                    .withOwnerReferences(createOwnerReference())
                     .endMetadata()
                     .withNewSpec()
-                        .withNewTo()
-                            .withKind("Service")
-                            .withName(perPodServiceName)
-                        .endTo()
-                        .withNewPort()
-                            .withNewTargetPort(EXTERNAL_PORT)
-                        .endPort()
-                        .withNewTls()
-                            .withTermination("passthrough")
-                        .endTls()
+                    .withNewTo()
+                    .withKind("Service")
+                    .withName(perPodServiceName)
+                    .endTo()
+                    .withNewPort()
+                    .withNewTargetPort(EXTERNAL_PORT)
+                    .endPort()
+                    .withNewTls()
+                    .withTermination("passthrough")
+                    .endTls()
```

Comment:
In general there are a lot of unrelated whitespace changes in this file, and while most of them are improvements it makes for a noisier diff when reviewing, so is best avoided in a PR which is ostensibly about something else. If it bugs you by all means open a separate PR for whitespace fixes. In this particular case (and others like it) the indentation really helps in seeing the YAML structure that the expression is building, so I'd rather we didn't remove it.

Reply:
Totally agree, I did do the indenting on some and not others, I'll fix this.

Reply:
Moral of the story, do not accidentally activate Intellij's Auto-Format option :-S